src/Silverback.Integration/Messaging/Messages/DefaultMessageHeaders.cs

```diff
@@ -99,5 +99,11 @@ public static class DefaultMessageHeaders
         ///     consumer side to determine the correct key to be used to decrypt the message.
         /// </remarks>
         public const string EncryptionKeyId = "x-encryption-key-id";
+
+        /// <summary>
+        ///     This will be set by the Move error policy and will contain the reason the failed
```

Comment:
Adjust summary according to other comment.

Reply:
Summaries fixed.